egs/librispeech/s5/local/chain/run_tdnn.sh

```diff
@@ -18,7 +34,6 @@ remove_egs=false
 common_egs_dir=
 xent_regularize=0.1
 self_repair_scale=0.00001
```

Comment:
Could you do me a favor and modify the self_repair_scale option name in the make_tdnn_configs.py to self-repair-scale-nonlinearity, to remain consistent with make_lstm_configs.py ? This would involve modifying all the tdnn scripts in all recipes, but grep and sed will simplify the process significantly.